docs/source/developers/guide/step_by_step/pr_and_github.rst

```diff
@@ -127,6 +127,49 @@ If all is set, you can make the pull request!
       and other additional information :ref:`pull_request_and_review`
       section.
 
+Continuous Integration (CI)
+---------------------------
+
+Continuous integration (CI) is an automated way to assert the code
```

Comment:
May want rephrasing; what are we asserting?

Reply:
Rephrased with using "run tests and builds on different environments" instead of "assert". Hope this makes more sense.